cmd/server-main.go

```diff
@@ -80,9 +80,9 @@ EXAMPLES:
 // Check for updates and print a notification message
 func checkUpdate(mode string) {
 	// Its OK to ignore any errors during getUpdateInfo() here.
-	if older, downloadURL, err := getUpdateInfo(1*time.Second, mode); err == nil {
-		if older > time.Duration(0) {
-			log.Println(colorizeUpdateMessage(downloadURL, older))
+	if currentReleaseTime, latestReleaseTime, downloadURL, err := getUpdateInfo(1*time.Second, mode); err == nil {
+		if updateMsg, isOld := computeUpdateMessage(downloadURL, currentReleaseTime, latestReleaseTime); isOld {
```

Comment:
You could do the same with `older` too.
```
t1 := UTCNow()
t2 := t1.Add(-older)
humanize.RelTime(t1, t2)
```

As returned values are not used in this function, I would prefer `older`.  Moreover returning multiple arguments looks ugly to me in this case.